system-int-test/test/cmr/system_int_test/search/scrolling_search_test.clj

```diff
@@ -0,0 +1,128 @@
+                                                                          coll1-cid 
+                                                                          {:granule-ur "Granule2"}))
+        gran3 (data2-core/ingest "PROV1" 
+                          (data2-granule/granule-with-umm-spec-collection coll1 
+                                                                          coll1-cid 
+                                                                          {:granule-ur "Granule3"}))
+        gran4 (data2-core/ingest "PROV1" 
+                          (data2-granule/granule-with-umm-spec-collection coll2 
+                                                                          coll2-cid 
+                                                                          {:granule-ur "Granule4"}))
+        gran5 (data2-core/ingest "PROV1" 
+                          (data2-granule/granule-with-umm-spec-collection coll2 
+                                                                          coll2-cid 
+                                                                          {:granule-ur "Granule5"}))
+        all-grans [gran1 gran2 gran3 gran4 gran5]]
+    (index/wait-until-indexed)
+               
+    (testing "Scrolling with page size"
+      (let [{:keys [hits scroll-id] :as result} (search/find-refs :granule {:provider "PROV1" :scroll true :page-size 2})]
+        (testing "First call returns scroll-id and hits count with page-size results"
+          (is (= (count all-grans) hits))
+          (is (not (nil? scroll-id)))
+          (is (data2-core/refs-match? [gran1 gran2] result)))
+        
+        (testing "Subsequent searches gets page-size results"
+          (let [result (search/find-refs :granule {:scroll true :scroll-id scroll-id})]
+            (is (= (count all-grans) hits))
+            (is (data2-core/refs-match? [gran3 gran4] result))))
+
+        (testing "Remaining results returned on last search"
+          (let [result (search/find-refs :granule {:scroll true :scroll-id scroll-id})]
+            (is (= (count all-grans) hits))
+            (is (data2-core/refs-match? [gran5] result))))
+
+        (testing "Searches beyond total hits return empty list"
+          (let [result (search/find-refs :granule {:scroll true :scroll-id scroll-id})]
+            (is (= (count all-grans) hits))
+            (is (data2-core/refs-match? [] result))))))
+
+    ;; The following test is included for completeness to test session timeouts, but it
+    ;; cannot be run regularly because it is impossible to predict how long it will take
+    ;; for Elasticsearch to actually time out a scroll session. Even when the scroll timeout
+    ;; is set to 1 second it may be many seconds before Elasticsearch disposes of the session. 
+    ;; The following test should not be removed and only uncommented during manual testing.
+    
+    ; (testing "Expired scroll-id is invalid"
```

Comment:
What about using #_ to comment the whole thing out instead? I know Parinfer doesn't understand #_ so you have to do some manual indenting to make things work.